aten/src/ATen/native/cpu/SumKernel.cpp

```diff
@@ -11,8 +11,35 @@ namespace at {
 namespace native {
 namespace {
 
+// Load vector from a smaller type to a larger type, reducing neighboring
+// elements until it fits into the vector size.
```

Comment:
This comment is a bit misleading, the smaller type element already fits into the vector size without adding two of them. Something like "Load vector with elements of a smaller type while performing pairwise reductions to fill the space of the vector's type", although even this could be improved.

Reply:
I've tried to clarify that I'm talking about the number of elements, not the size of that element:
```
// Load vector from a smaller type (more elements) to a larger type (fewer elements),
```

Does that clear up your confusion?

Reply:
It does, thank you!